### ISSUE TITLE
Back button doesnt work when going from normal page to databrowser

### ISSUE THREAD

**Issue by melvincarvalho** [linkeddata/mashlib#14]
## Tested on  

Version 64.0.3282.167 (Official Build) Built on Ubuntu , running on Ubuntu 16.04 (64-bit)

## To reproduce

Go to :  https://drive.verborgh.org/public/yo/
Paste into browser : https://drive.verborgh.org/public/yo/data/yo.ttl

## Expected behaviour

Browser should return back to https://drive.verborgh.org/public/yo/

## Actual behaviour

Browser stays on : https://drive.verborgh.org/public/yo/data/yo.ttl

## Diagnosis

Related [code](https://github.com/linkeddata/mashlib/blob/90b3ae9409d4784fc236e34ffe13ca26f36543be/index.js#L27) : 

```JavaScript
window.onpopstate = function(event) {
  window.document.outline.GotoSubject($rdf.sym(window.document.location.href), true, undefined, true, undefined)
}
```

Currently the assumption is that a previous page would also be called from the databrowser.  It would be better if the onpopstate handler was more aware of the type of page that called it.

**Comment by timbl** [comment]
True. Maybe the databrowser leaves a trace property in the event, like paneName .
If n pane name,  allow the normal browser to reload the page.
